core/src/main/java/org/frankframework/util/RunStateManager.java

```diff
@@ -34,8 +34,7 @@ public synchronized RunState getRunState() {
 
 	public synchronized void setRunState(RunState newRunState) {
 		if(runState != newRunState) {
-			if(log.isDebugEnabled())
-				log.debug("Runstate [" + this + "] set from " + runState + " to " + newRunState);
+			log.debug("Runstate [{}] set from {} to {}", super.toString(), runState, newRunState);
```

Comment:
Fixes double current state (inherited from `toString`):
`util.RunStateManager - Runstate [RunStateManager@75a9bbef [STOPPED]] set from STOPPED to STARTING`